client/db.go

```diff
@@ -469,7 +469,7 @@ func (db *DB) Txn(retryable func(txn *Txn) error) error {
 // returns any errors.
```

Comment:
Update the comment to say that it returns nil rather than an empty response for empty input.